ui/src/pages/style.css

```diff
@@ -18,6 +18,10 @@ body {
     min-width: 960px;
 }
 
+.grey_background {
+    background: #f2f4f5; /* move to class to applied changes only for Monitoring Dashboard */
```

Comment:
Please use splunk theme guidelines. We don't what to handle dark/light theming by ourself

Reply:
same as for adding that class